src/string/trimEnd.ts

```diff
@@ -16,20 +18,11 @@ export function trimEnd(str: string, chars?: string | string[]): string {
     return str.trimEnd();
   }
 
+  const charsToUse = Array.isArray(chars) ? flatMap(chars, char => char.split('')) : chars;
```

Comment:
Handle `chars` inside `trimStart/trimEnd` by consolidating them, `switch statements` can also be removed.